js_modules/dagster-ui/packages/ui-core/worker-typings.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * A list of dependencies that are disallowed in WebWorkers.
+ */
+
+declare module '@vx/shape';
+
+declare module 'react' {}
+declare module 'react-dom' {}
```

Comment:
List the modules that we should guard against. These allow us to surface errors when `React` imports are used.

Reply:
Lets also add apollo/client